v2/group_type.go

```diff
@@ -10,3 +10,9 @@
 package v2
 // GroupType : The group type.
 type GroupType string
+
+// List of GroupType
+const (
```

Comment:
This shouldn't be there. Please run the sed commands in the CONTRIBUTING section and re-commit this file